### PR TITLE
Update meta descriptions

### PR DESCRIPTION
**Pull request by helenktsai**
This PR is a first step towards improving the meta descriptions on our site

### CODE REVIEW

**Review by soujay — APPROVED** [review]

File: nservicebus/sagas/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Sagas
-summary: NServiceBus uses event-driven architecture to include fault-tolerance and scalability in long-term business processes.
+summary: Maintain statefulness in distributed systems with the saga pattern and event-driven architectural principles with built-in fault-tolerance and scalability.
```
Comment (tamararivera) [inline nservicebus/sagas/index.md:3]: Maybe we should mention NServiceBus here too

**Review by tamararivera — APPROVED** [review]

File: nservicebus/sagas/index.md (hunk 1 above)
Comment (helenktsai) [reply nservicebus/sagas/index.md:3]: It's already at 155 chars, so have to sacrifice something else. Do you have a suggestion?

Comment (helenktsai) [inline nservicebus/sagas/index.md:3]: How's about

```suggestion
summary: Maintain statefulness in distributed systems with the saga pattern and NServiceBus' event-driven architecture with built-in fault-tolerance and scalability.
```

?

Comment (tamararivera) [reply nservicebus/sagas/index.md]: I like it!

**Pull request merged**